modules/misc/news.nix

```diff
@@ -448,6 +448,20 @@ in
           A new program module is available: 'programs.neovim'.
         '';
       }
+      {
+        time = "2017-11-14T19:56:49+00:00";
+        condition = with config.xsession.windowManager; (
+          i3.enable && i3.config != null && i3.config.startup != []
+        );
+        message = ''
+          A new 'notification' option was added to
+          config.xsession.windowManager.i3.startup submodule.
```

Comment:
I think you can remove the `config.` in `config.xsession.windowManager.i3.startup`.